### PR TITLE
Register new package PolyChaos.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [timueh/PolyChaos.jl](https://github.com/timueh/PolyChaos.jl)
Release: [v0.1.0](https://github.com/timueh/PolyChaos.jl/releases/tag/v0.1.0)
Travis: [Travis Build Status](https://travis-ci.org/timueh/PolyChaos.jl/builds/490532410)
cc: @timueh

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

@timueh This PR will remain open for three days for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by dpsanders** [comment]
Maybe change the name to PolynomialChaos.jl?

**Comment by timueh** [comment]
> Maybe change the name to PolynomialChaos.jl?

Well, when I started writing the package, the sole purpose was to have a Julia package for polynomial chaos, yes, and the name was `PolynomialChaos.jl`. But in the process of writing it, it more became a package that combines orthogonal polynomials, quadrature rules, and has polynomial chaos as a *specific* application. So technically, the eventual ideal solution would be to encapsulate the polynomial-chaos-functionalities in a separate package. But for now that is too much overhead. I would make that dependent on whether the package is actually being used by someone else but myself ;)

Having this in mind, names like `OrthoQuadPolynomialChaos.jl` popped up, which I rejected for obvious reasons.. 😃

**Comment by dpsanders** [comment]
Are you aware of the varios other packages for orth polys, in particular ApproxFun.jl?

**Comment by timueh** [comment]
I have heard of `ApproxFun.jl`, yes. It is similar to `chebfun` for Matlab, and touches a different aspect of orthogonal polynomials, namely approximation with its various applications such as solving ODEs/PDEs.

The basic task for `PolyChaos.jl` is the following: given a measure $m$, compute the polynomials orthogonal to $m$. (Once we know the orthogonal polynomials, we can also compute quadrature rules and use it for polynomial chaos expansions.) To the best of my knowledge there does not exist a package that does that.

Looking at juliaobserver.com I found the following packages related to orthogonal polynomials:
- `Jacobi.jl`, which is restricted to Jacobi polynomials.
- `Orthopolys.jl`, which supports Jacobi polynomials, Gegenbauer polynomials,  Hermite polynomials, but hasn't been touched in a year.
- `FastTransforms.jl`, which covers polynomial transformations.

For quadrature rules, there are `FastGaussQuadrature.jl` and `GaussQuadrature.jl` both of which are restricted to the quadrature rules.

To summarize, `PolyChaos.jl` let's you input some arbitrary measure, then it computes the respective orthogonal polynomials. If you wish, you can get the respective quadrature rule (Gauss, Gauss-Radau, Gauss-Lobatto). If the measure happens to be a probability density, you can apply polynomial chaos expansions. Of course, the usual suspects such as Jacobi, Hermite, and so on are implemented because there are closed-form expressions for the recurrence coefficients of the orthogonal polynomials.

**Comment by timueh** [comment]
just wondering: what is the usual time to accept/reject the pull requests? thx!
